src/cmds/qstat.c

```diff
@@ -1215,34 +1215,43 @@ display_statjob(struct batch_status *status, struct batch_status *prtheader, int
 	char *cmdargs = NULL;
 	char *hpcbp_executable;
 
-	if (how_opt & ALT_DISPLAY_INCR_WIDTH) {
+	if (wide) {
+		sprintf(format, "%%-%ds %%-%ds %%-%ds  %%%ds %%%ds %%-%ds\n",
+			SIZEJOBID, SIZEJOBNAME, SIZEUSER, TIMEUL, STATEL, PBS_MAXQUEUENAME);
+	} else if (how_opt & ALT_DISPLAY_INCR_WIDTH) {
 		sprintf(format, "%%-%ds %%-%ds %%-%ds  %%%ds %%%ds %%-%ds\n",
 			PBS_MAXSEQNUM+10, NAMEL, OWNERL, TIMEUL, STATEL, LOCL);
 	} else {
 		sprintf(format, "%%-%ds %%-%ds %%-%ds  %%%ds %%%ds %%-%ds\n",
 			PBS_MAXSEQNUM+5, NAMEL, OWNERL, TIMEUL, STATEL, LOCL);
 	}
 
-	if (! full && prtheader && output_format == FORMAT_DEFAULT) {
+	if (!full && prtheader && output_format == FORMAT_DEFAULT) {
 		c = get_attr(prtheader->attribs, ATTR_comment, NULL);
 		if (c)
 			printf("%s\n", show_nonprint_chars(c));
 		if (how_opt & ALT_DISPLAY_p) {
-				if (how_opt & ALT_DISPLAY_INCR_WIDTH) {
-					printf("Job id                 Name             User               %% done  S Queue\n");
-					printf("---------------------  ---------------- ----------------  -------- - -----\n");
-				} else {
-					printf("Job id            Name             User               %% done  S Queue\n");
-					printf("----------------  ---------------- ----------------  -------- - -----\n");
-				}
+			if (wide) {
+				printf("Job id                         Name            User              %% done  S Queue          \n");
+				printf("-----------------------------  --------------- ---------------  -------- - ---------------\n");
```

Comment:
I guess it doesn't matter, but why are there spaces after the world 'Queue' and before the newline?

Reply:
I can remove them.